client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestEditPage.tsx

```diff
@@ -52,12 +70,9 @@ export const RequestLineEditPage = () => {
     });
   }, [currentItem]);
 
-  if (isLoading || requestIsLoading) return <BasicSpinner />;
```

Comment:
I think important to note - it was just the `isLoading` here that was causing the issue - unmounting the entire page whenever the `useDraftRequisitionLine` loading state was true, which was every time we saved. However I like the introduction of the `Inner` component, let's keep it 😁